.devcontainer/devcontainer.json

```diff
@@ -1,13 +1,18 @@
 {
   "name": "Bicep Development Environment",
-  "extensions": [
-    "msazurermtools.azurerm-vscode-tools",
-    "ms-azuretools.vscode-bicep",
-    "ms-dotnettools.csharp",
-    "ms-azuretools.vscode-docker",
-    "editorconfig.editorconfig",
-    "github.vscode-pull-request-github"
-  ],
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
```

Comment:
Shouldn't this be `mcr.microsoft.com/vscode/devcontainers/universal:2`?

Reply:
```suggestion
  "image": "mcr.microsoft.com/vscode/devcontainers/universal:2",
```